<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RingProperties</class>
 <widget class="QWidget" name="RingProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ring Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="UtQtLonPosLineEdit" name="mLonEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="mName">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="UtQtColorButton" name="mRingColor">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mLatLabel">
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="UtQtColorButton" name="mRangeLabelColor">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="mRingLineWidth">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="UtQtLengthValueEdit" name="mCenterRadius"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="mRangeLabels">
     <property name="text">
      <string>Range Labels</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="mAdditionalRings">
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Additional Radii</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="UtQtLengthValueEdit" name="mAdditionalRadii">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="mAngleLabels">
     <property name="text">
      <string>Angle Labels</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="UtQtColorButton" name="mAngleLabelColor">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Additional Rings</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Ring Color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mLonLabel">
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Ring Line Width</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Center Radius</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QCheckBox" name="mAlignWithPlatformCheckBox">
     <property name="text">
      <string>Align with Platform</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Radial Rate</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Angle Width</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="UtQtAngleValueEdit" name="mCenterAngle"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Radial Color</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Center Angle</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="UtQtAngleValueEdit" name="mRadialRate"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Radial Line Width</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="UtQtColorButton" name="mRadialColor">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="UtQtAngleValueEdit" name="mAngleWidth"/>
   </item>
   <item row="11" column="1">
    <widget class="QSpinBox" name="mRadialLineWidth">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="UtQtLatPosLineEdit" name="mLatEdit"/>
     </item>
     <item>
      <widget class="wkf::LocationSelector" name="mLocationSelector">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="17" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wkf::LocationSelector</class>
   <extends>QPushButton</extends>
   <header>WkfSelectorWidgets.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLatPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLonPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLengthValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtAngleValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtColorButton</class>
   <extends>QPushButton</extends>
   <header>UtQtColorButton.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mName</tabstop>
  <tabstop>mRangeLabels</tabstop>
  <tabstop>mRangeLabelColor</tabstop>
  <tabstop>mRingColor</tabstop>
  <tabstop>mRingLineWidth</tabstop>
  <tabstop>mCenterRadius</tabstop>
  <tabstop>mAdditionalRings</tabstop>
  <tabstop>mAdditionalRadii</tabstop>
  <tabstop>mAngleLabels</tabstop>
  <tabstop>mAngleLabelColor</tabstop>
  <tabstop>mRadialColor</tabstop>
  <tabstop>mRadialLineWidth</tabstop>
  <tabstop>mAlignWithPlatformCheckBox</tabstop>
  <tabstop>mCenterAngle</tabstop>
  <tabstop>mAngleWidth</tabstop>
  <tabstop>mRadialRate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
